dist/api_data.py

```diff
@@ -1137,6 +1137,12 @@ def __ge__(self, other):
     wiredtiger_open_log_configuration +\
     wiredtiger_open_tiered_storage_configuration +\
     wiredtiger_open_statistics_log_configuration + [
+    Config('backup_partial_restore', 'false', r'''
+        When set to true, allow a partial restore on the current database using the
+        WiredTiger metadata backup file. WiredTiger will reconstruct the
+        database files that are only present in the directory and remove all
+        table entries that are not in the directory anymore.''', 
```

Comment:
Nit: Do not end the description with a period '.' because the auto-generator adds one automatically.